src/main/java/uk/gov/hmcts/reform/prl/services/ManageOrderService.java

```diff
@@ -138,6 +187,18 @@ private Map<String,String> getOrderTemplateAndFile(CreateSelectOrderOptionsEnum
                 fieldsMap.put(PrlAppsConstants.FINAL_TEMPLATE_NAME, c21Template);
                 fieldsMap.put(PrlAppsConstants.GENERATE_FILE_NAME, c21File);
                 break;
+            case transferOfCaseToAnotherCourt:
+                fieldsMap.put(PrlAppsConstants.TEMPLATE, c49TDraftTemplate);
+                fieldsMap.put(PrlAppsConstants.FILE_NAME, c49DraftFile);
+                fieldsMap.put(PrlAppsConstants.FINAL_TEMPLATE_NAME, c49Template);
+                fieldsMap.put(PrlAppsConstants.GENERATE_FILE_NAME, c49File);
+                break;
+            case powerOfArrest:
+                fieldsMap.put(PrlAppsConstants.TEMPLATE, c49TDraftTemplate);
```

Comment:
PowerOfArrest order template is different, its not c49. Please refer templates given in https://tools.hmcts.net/jira/browse/PRL-1313

Reply:
@cshiwarale we placed this as dummy template will replace it with actual template once that is ready..